styles/header.css

```diff
@@ -75,3 +148,16 @@
     }
   }
 }
+
+.dropdown-menu {
+  --bs-dropdown-padding-x: 0.125rem;
+  --bs-dropdown-padding-y: 0.125rem;
+  --bs-dropdown-item-padding-x: 0.5rem;
+  --bs-dropdown-item-padding-y: 0.125rem;
+
+  &.show {
+    display: flex;
```

Comment:
Why do we need this? I'd prefer we don't needlessly override bootstrap.  How is this different than `display: block`?

Reply:
This is what fixes https://github.com/sul-dlss/component-library/issues/92.

Reply:
This selector touches every `.dropdown-menu`.  Should we just be concerned about those in the `.masthead .navbar`?

Reply:
Updated the selector to scope it to `.navbar`.